bin/reth/src/args/rpc_server_args.rs

```diff
@@ -132,6 +135,10 @@ pub struct RpcServerArgs {
     #[arg(long, value_name = "COUNT", default_value_t = RPC_DEFAULT_MAX_TRACING_REQUESTS)]
     pub rpc_max_tracing_requests: u32,
 
+    /// Maximum gas limit for `eth_call` and call tracing RPC methods.
+    #[arg(long, default_value_t = RPC_DEFAULT_GAS_CAP)]
```

Comment:
needs an alias for

> --rpc.gascap value             (default: 50000000)

Reply:
agree, ref: https://github.com/ethereum/go-ethereum/blob/6ca3ef9a7b7ebf3ed0004cfdca0c7a27a38c28e7/cmd/utils/flags.go#L539-L544